gsoc/2017.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: page
+title: "Google Summer of Code 2017 Scala Projects"
+---
+
+*This page is work in progress for the upcoming GSOC 2017*
+
+## Google Summer of Code
+
+**This year the Scala team applied again for the [Google Summer of
+  Code](https://developers.google.com/open-source/gsoc/)
+  program to work with enthusiastic students on challenging Scala
+  projects**
+
+This page provides a list of project ideas. The suggestions are only a
+starting point for students. We expect students to explore the ideas
+in much more detail, preferably with their own suggestions and
+detailed plans on how they want to proceed. Don't feel constrained by
+the provided list! We welcome any of your own challenging ideas, but
+make sure that the proposed project satisfies the [main
+requirements](#requirements-and-guidelines) mentioned below.
+
+### How to get involved
+
+The best place to propose and discuss your proposals is our
+["contributors"](http://contributors.scala-lang.org/)
+discussion forum. This way you will get quickly responses from the whole
+Scala community.
+
+### Previous Summer of Code
+
+We encourage you to have a look at our
+[2016](http://scala-lang.org/gsoc/2016.html),
+[2015](http://scala-lang.org/gsoc/2015.html),
+[2014](http://scala-lang.org/gsoc/2014.html),
+[2013](http://www.scala-lang.org/news/2013/03/20/gsoc13.html),
+[2012](http://www.scala-lang.org/gsoc2012#),
+[2011](http://www.scala-lang.org/gsoc2011#),
+[Summer of Code 2010](http://www.scala-lang.org/gsoc2010#)
+pages to get an idea on what we and you can expect while working on Scala.
+
+## Project Ideas
+
+Here are some project ideas. The list is non-binding and any reasonable
+project related to Scala that is proposed by a student will be thoroughly
+reviewed.
+
+*... Mentors: please insert your projects here. You can use the following template ...*
+
+### *Project name*
+
+*Project description.*
+
+*Link to the corresponding code repository, if relevant.*
+
+Supervised by *[@username](https://github.com/username)*.
+
+
+## Requirements and Guidelines
+
+### General Student Application Requirements
+
+This is the sixth time the Scala project has applied to the Summer of
+Code, and from last years experience, increased popularity of the
+language and stories of other mentor organizations we expect a high
+number of applications. First, be aware of the following:
+
+*   Make sure that you understand, fulfill and agree to the general
+    [Google Summer of Code rules](https://summerofcode.withgoogle.com/rules/)
+*   The work done during GSoC requires some discipline from the
+    students as they have to plan their day-to-day activities by
```

Comment:
First point was aimed at "you", second is aimed at "the student". I suggest we change that to "you".